### PR TITLE
First pass of easter egg work

### PR DESCRIPTION
**Pull request by brunnerjosh**

### CODE REVIEW

File: packages/react-scripts/template/src/Styles/Button.styl [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  font-size: 1em
+  color: $primary
+  cursor: pointer
+  padding: .5em 1em
+  background: $white
+  border-radius: rem(5);
+  transition: background 250ms ease
+
+  &:hover
+    background: darken($white, 15)
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Button.styl]: Remove this file.

File: packages/react-scripts/template/src/Components/Landing.js [hunk 2]
```diff
@@ -4,7 +4,10 @@ export default class Landing extends React.Component {
   render() {
     return (
       <div className="landing">
-        <p>‍👨‍🚀</p>
+        <section className="easter-egg">
+          <p className="easter-egg__spacheman">👨‍🚀</p>
+          <p className="easter-egg__rocket">🚀</p>
+        </section>
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Components/Landing.js]: rename `easter-egg` to `outerspace` , `easter-egg__spacheman` to `spacheman` and `easter-egg__rocket` to `rocket`

Comment (canfie1d) [reply packages/react-scripts/template/src/Components/Landing.js]: Misuse of <section> el.
"The <section> tag defines sections in a document, such as chapters, headers, footers, or any other sections of the document."

File: packages/react-scripts/template/src/Components/Landing.js [hunk 3]
```diff
@@ -4,7 +4,10 @@ export default class Landing extends React.Component {
   render() {
     return (
       <div className="landing">
-        <p>‍👨‍🚀</p>
+        <section className="easter-egg">
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Components/Landing.js]: Misuse of `<section>` el.
"The `<section>` tag defines sections in a document, such as chapters, headers, footers, or any other sections of the document."

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 4]
```diff
@@ -34,3 +38,44 @@
     }
   }
 }
+
+.easter-egg
+  display: flex
+  align-items: center
+  justify-content: center
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: Use mixin

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 5]
```diff
@@ -34,3 +38,44 @@
...
+
+  &__spacheman
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: Use complete class names

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 6]
```diff
@@ -34,3 +38,44 @@
...
+    color: black                                        // Satisfies aXe warning
+    position: fixed
+    perspective: 1000px;
+    transition: 4s linear
+    transform: rotate(-20deg)
+    backface-visibility: hidden;
+    animation: fly 4s linear infinite;
+
+@keyframes fly {
+  20%, ,40%, 60%, 80% {
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: extra `,`

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 7]
```diff
@@ -34,3 +38,44 @@
...
+  &__rocket
+    left: 0vw
+    top: 110vh
+    z-index: 1
+    color: black                                        // Satisfies aXe warning
+    position: fixed
+    perspective: 1000px;
+    transition: 4s linear
+    transform: rotate(-20deg)
+    backface-visibility: hidden;
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: I don't think we need `backface-visibility` or `perspective`

**Review by canfie1d — CHANGES_REQUESTED** [review]
Coupla changes

File: packages/react-scripts/template/src/Styles/Button.styl (hunk 1 above)
Comment (brunnerjosh) [reply packages/react-scripts/template/src/Styles/Button.styl]: It's removed in #11

File: packages/react-scripts/template/src/Styles/Landing.styl (hunk 6 above)
Comment (brunnerjosh) [reply packages/react-scripts/template/src/Styles/Landing.styl]: Nice catch

File: packages/react-scripts/template/src/Styles/Landing.styl (hunk 7 above)
Comment (brunnerjosh) [reply packages/react-scripts/template/src/Styles/Landing.styl]: Another nice catch. This was leftover from a previous idea  :/

File: packages/react-scripts/template/src/Styles/Landing.styl (hunk 4 above)
Comment (brunnerjosh) [reply packages/react-scripts/template/src/Styles/Landing.styl]: I love that mixin!

File: packages/react-scripts/template/src/Components/Landing.js (hunk 3 above)
Comment (brunnerjosh) [reply packages/react-scripts/template/src/Components/Landing.js]: Good thinking. I've switched it to a div and reset unneeded styles.

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 8]
```diff
@@ -11,6 +11,18 @@
 
   button {
     margin: .5em
+    border: none
+    font-size: 1em
+    cursor: pointer
+    padding: .5em 1em
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: Use rems for all expect for text.

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 9]
```diff
@@ -11,6 +11,18 @@
 
   button {
     margin: .5em
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: Use rems

Comment (canfie1d) [reply packages/react-scripts/template/src/Styles/Landing.styl]: `rem(8)`

File: packages/react-scripts/template/src/Styles/Landing.styl (hunk 8 above)
Comment (canfie1d) [reply packages/react-scripts/template/src/Styles/Landing.styl]: `rem(8 16)`

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 10]
```diff
@@ -11,6 +11,18 @@
...
+    font-size: 1em
+    cursor: pointer
+    padding: .5em 1em
+    background: $white
+    displayFontBold();
+    border-radius: rem(5);
+    transition: background 250ms ease
+
+    &:hover {
+      background: darken($white, 15)
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: There are gray color variables available

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 11]
```diff
@@ -37,41 +49,50 @@
...
-}
 
-.easter-egg
-  display: flex
-  align-items: center
-  justify-content: center
+  .outerspace {
+    margin: initial
+    padding: initial
+    border-radius: initial
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: What are these 3 line accomplishing?

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 12]
```diff
@@ -37,41 +49,50 @@
...
+    background: none
+    flexContainer(true)
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: Use layout functions first

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 13]
```diff
@@ -37,41 +49,50 @@
...
 
-  &__spacheman
-    cursor: pointer
-    user-select: none
-    transition: transform 250ms ease
-    transform: scale(1)
-    transition-delay: 3.4s
+    &:before {
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: Double colon here -> `&::before`

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 14]
```diff
@@ -37,41 +49,50 @@
...
+    }
 
-    &:hover
-      transform: scale(0)
+    .spacheman {
+      cursor: pointer
+      user-select: none
+      transition: transform 250ms ease
+      transform: scale(1)
+      transition-delay: 3.4s
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: Use `3400ms`

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 15]
```diff
@@ -37,41 +49,50 @@
...
+      }
+    }
+
+    .rocket {
+      left: 0vw
+      top: 110vh
+      z-index: 1
+      color: black                                        // Satisfies aXe warning
+      position: fixed
+      transition: 4s linear
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl]: Use `4000ms` & define property

**Review by canfie1d — CHANGES_REQUESTED** [review]

File: packages/react-scripts/template/src/Styles/Landing.styl (hunk 11 above)
Comment (brunnerjosh) [reply packages/react-scripts/template/src/Styles/Landing.styl]: They are resetting the styles on the `div` element in `Landing.styl`

**Review by canfie1d — APPROVED** [review]

**Pull request merged**